core/include/detray/tools/navigator.hpp

```diff
@@ -1,17 +1,23 @@
 /** Detray library, part of the ACTS project (R&D line)
  *
- * (c) 2021 CERN for the benefit of the ACTS project
+ * (c) 2021-2022 CERN for the benefit of the ACTS project
  *
  * Mozilla Public License Version 2.0
  */
 
 #pragma once
 
+#if defined(__CUDACC__)
+#include <thrust/sort.h>
+#endif
```

Comment:
Unlike `thrust/tuple.h`, it seems `thrust/sort.h` cannot be called by host code. Let me find a better way though

Reply:
Just added `sequenctial_sort` function in `accessor.hpp` as a temporary solution

Reply:
:thinking: Okay, I played a bit. I **think** it will be possible to make the code use Thrust everywhere for things that it needs Thrust for on the device.

But it will be a bit more complicated to do that I first thought. So that will have to be handled in a separate PR. For now let's just keep this dependency on `__CUDACC__` in the code.